src/config.json

```diff
@@ -59,7 +59,7 @@
         "phantomjs": "1.9.18",
         "grunt-lib-phantomjs": "0.3.0",
         "grunt-eslint": "19.0.0",
-        "grunt-contrib-watch": "0.4.3",
```

Comment:
@Blackweda Sorry if I am missing something here, but is the change `"grunt-contrib-watch": "1.0.0"` is intentional for this PR ?